7.5.цикл_while:_обработка_цифр_числа/7.5.6.py

```diff
@@ -0,0 +1,11 @@
+n = int(input())         
+ma = 0                 
+mi = 9                 
+while n != 0:            
+    if n % 10 > ma:    
+        ma = n % 10    
+    if n % 10 < mi:    
+        mi = n % 10    
+    n = n // 10          
+print(f"Максимальная цифра равна {ma}")
+print(f"Минимальная цифра равна {mi}") 
```

Comment:
Что используешь f-строки - молодец, но опять же, чем меньше вызовов одинаковых функций - тем лучше:
```suggestion
print(f"Максимальная цифра равна {ma}", f"Минимальная цифра равна {mi}", sep="\n")
```